style.css

```diff
@@ -509,13 +442,325 @@ textarea {
 }
 
 .msg-field {
-  font-family: 'Inter', sans-serif;
   font-style: normal;
-  font-weight: normal;
   font-size: 15px;
   line-height: 24px;
 }
 
 .msg-field::placeholder {
   color: #81818c;
 }
+
+/* Id Selectors */
+#p-headline {
+  font-size: 20px;
+  line-height: 28px;
+  color: #c8c8d4;
+  width: 327px;
+  height: 170px;
+}
+
+#documentation {
+  background-image: url(img/ellipse19.svg);
+  background-repeat: no-repeat;
+  background-position: right top;
+}
+
+#arrow-down {
+  padding: 0 1em;
+}
+
+#arrow-down1 {
+  padding: 0 1em;
+}
+
+#form-section {
+  border-bottom: 2px solid #3a3837;
+  padding-bottom: 4em;
+}
+
+/* -----Hidden elements----- */
+.hidden-el {
+  display: none;
+}
+
+/* -----Animations and Transitions----- */
+.fade-in {
+  opacity: 1;
+  animation-name: fadeInOpacity;
+  animation-iteration-count: 1;
+  animation-timing-function: ease-in;
+  animation-duration: 2s;
+}
+
+@keyframes fadeInOpacity {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+/* -----Desktop media query Version----- */
+@media screen and (min-width: 992px) and (max-width: 1400px) {
```

Comment:
- I realized what the problem with your project was and it lies in this line. What you are doing here, is declaring that whenever the screen has a minimum width of 992px, the upcoming rules would apply, but if the screen is bigger than 1400px, the rules applied are going to be the same that is applied to the mobile version. You have to be careful with this kind of media queries. Right now, I am using a screen that is 1920px width, so all the rules that you set for the desktop version below this line were not being applied. I recommend avoiding this kind of limit if you are not planning to give a different design to bigger screens, otherwise, it is going to keep giving you problems like these ones.